NuEVI/menu.cpp

```diff
@@ -501,22 +501,35 @@ const MenuEntrySub specialKeyMenu = {
   , nullptr
 };
 
+uint16_t wireless_power=0;
+uint16_t wireless_channel=4;
```

Comment:
these could be made static

Reply:
if you say so. i never get the difference.